### ISSUE TITLE
Apple 1 softlist basic tapes are invalid (but here's one that works)

### ISSUE THREAD

**Issue by mcfilmmakers** [mamedev/mame#9240]
I apologize if this isn't the correct place to post this issue as this is my first time interacting with the "backend" of MAME dev. Anyway, I've come to notice that neither the Apple 1 basic nor Apple 1 basica files are invalid.

Some games (such as Acey Ducey) requires loading the basic tape first before being able to load the game tape. To do so, you load the basic tape then type C100R, return, E000.EFFFE, return, then start the tape. Once the tape finishes loading, you need to type E000R then return.

At this point, the machine is supposed to respond with E000:4C. Instead, it respond with E000:FF. That response means the tape is no good.

However, the basic tape found at this location: https://archive.org/download/Apple1Cassettes DOES work. The problem is, I can't load that tape via LUA because it isn't recognized by MAME (LUA outputs in the console a list of supported software items (best match first).

If you could integrate the properly working basic tape to the software list, that would be great!

**Comment by Tafoid** [comment]
While I don't have direct Apple 1 knowledge, regarding software loading, you should be able to load a piece of software not showing on the software list by loading as a supported media.  In apple1's case, you can use the -cass (-cassette) to load an image from a location on your storage.
> mame apple1 -cassette c:\location\to\aceyducey.wav (or .zip if zipped)

**Comment by mcfilmmakers** [comment]
You're right, I changed it to :load instead of :load_software and that did the trick. However, the better solution is replacing the image with a working one.

**Comment by colinhowell** [comment]
I remember dealing with issues loading the Apple I cassette BASIC, and differences between the images for that BASIC in the software list, several years back (spring of 2017, it turns out). You might want to take a look at these threads on bannister.org for further details:

[First thread: the finickiness of the "basic" tape image](https://forums.bannister.org/ubbthreads.php?ubb=showflat&Number=109478#Post109478). The image has poor signal quality, so you have to be careful how you load it to be sure the emulation reads it properly instead of rejecting it.

[Second thread: differences in the contents between the "basic" and "basica" images](https://forums.bannister.org/ubbthreads.php?ubb=showflat&Number=109605#Post109605). The "basica" image actually seems closer to the original version of Apple I BASIC, while the "basic" image seems to be from a patched version of that BASIC.

**Comment by mcfilmmakers** [comment]
So according to those threads the issue is cause by an overly sensitive disk reader in the MAME machine and/or a poorly recoded image.

As I pointed out, the image liked to works in the current machine as is without having to time yourself in a way that isn't true to end users of the real thing.

I don't have the ability to do anything about either issue other than to point it out to those that care to resolve it for end users. Good luck!
